corehq/apps/app_manager/templates/app_manager/partials/build_profiles_select.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-sm-3">
+    {% trans 'Build Profile' %}:
+    <select id="build-profile-select-for-multimedia" class="form-control">
```

Comment:
Can you make this a select2? It should just be a matter of adding the class `hqwebapp-select2` and making sure `hqwebapp/js/widgets` is available to this page.

Reply:
https://github.com/dimagi/commcare-hq/pull/26032/commits/91ae2fc17a7afcb3776d48fee16d192df21a734c